conf/ocp_version/ocp-4.4-upgrade.yaml

```diff
@@ -1,5 +1,4 @@
 ---
-# Use this config for upgrading of OCP 4.3 to 4.4 cluster
+# Use this config for upgrading of OCP cluster to latest 4.4 nightly build
 UPGRADE:
-  ocp_upgrade_version: "4.4.0-0.nightly"
```

Comment:
Why do we don't need this ocp_upgrade_version with "4.4.0-0.nightly" anymore?

Reply:
we should use `ocp_channel` 4.4-fast - it will automatically select latest nightly build

Reply:
What I understand the 4.4-fast is not nightly ;). What is in the fast channel is released one to fast channel but not nightly, we have also stable channel.  There are sometimes differences between fast and stable, but looks like now we have the same content in fast and stable channel:
```
$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.4' | jq -r '.nodes[] | .version' | sort -V | tail -1
4.4.14
$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=fast-4.4' | jq -r '.nodes[] | .version' | sort -V | tail -1
4.4.14

$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.5' | jq -r '.nodes[] | .version' | sort -V | tail -1
4.5.4
$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=fast-4.5' | jq -r '.nodes[] | .version' | sort -V | tail -1
4.5.4
```

But nightly build is https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/#4.5.0-0.nightly  something like: 4.5.0-0.nightly-2020-08-03-123303

Are you sure about what you are saying here about fast channel?